### PR TITLE
Prep for 1.2.0 release

### PR DESCRIPTION
**Pull request by aaabdelgany**

### CODE REVIEW

**Review by ggambetti — APPROVED** [review]
LGTM, does this need a changelog entry?

File: Base.gitlab-ci.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@ default:
     CONFIGURATION_VERSION_ID: $configuration_version_id
     MESSAGE: "Base template message. Override this"
```
Comment (mjyocca) [inline Base.gitlab-ci.yml:71]: Do we need to add `SAVE_PLAN` and `IS_DESTROY` here? or not necessary?

Comment (zainq11) [reply Base.gitlab-ci.yml:71]: I think we would need to do that if these variables were generated by intermediate stages or to override them. 
```
CONFIGURATION_VERSION_ID: $configuration_version_id
```
`$configuration_version_id` was set by an earlier stage which is why we have defined it here as a variable.
Are `SAVE_PLAN` and `IS_DESTROY` provided by the user directly?

Comment (aaabdelgany) [reply Base.gitlab-ci.yml:71]: That is correct. So we should be good as is if I'm understanding you correctly.

**Review by zainq11 — APPROVED** [review]

**Review by mjyocca — APPROVED** [review]
LGTM. Thanks @zainq11 and @aaabdelgany!!

**Pull request merged**